WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -125,7 +125,6 @@ decisions = {
 					NOT = { culture = ROOT }
 				}
 			}
-			holy_order = no
```

Comment:
Both are unrealistic. But you are not addressing the issue (culture conversion) by removing the existant checks. I prefer actually addressing it by tweaking the conversion rates.